ci/test_cpp.sh

```diff
@@ -9,7 +9,23 @@ rapids-logger "Generate C++ testing dependencies"
 rapids-dependency-file-generator \
   --output conda \
   --file_key test_cpp \
-  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch)" | tee env.yaml
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch)" | tee test.yaml
+
+rapids-logger "Downloading artifacts from previous jobs"
+CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
+
+cat <<EOF | tee cuml.yaml
+name: cuml
+channels:
+  - ${CPP_CHANNEL}
+dependencies:
+  - libcuml
+  - libcuml-tests
+EOF
+
+# TODO: install conda-merge in the CI images
+pip install conda-merge
```

Comment:
Small question: why use pip for installing conda-merge?

Reply:
It's fastest to do it this way for the prototype. I am installing this into the conda environment in https://github.com/rapidsai/ci-imgs/pull/112.

Reply:
We may also want to add `conda-merge` to the base image if we find it useful. Could imagine this pattern showing up in other projects' CI